prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Dre Sean';
+    var numberOfPets = 1;
+    var likesVegetables = true;
+    var nothing = 'null';
```

Comment:
Hi Dre, here you want to leave out the quotes so that JavaScript interprets the value as `null` and not a string.